### DISCUSSION TITLE
Is it possible to reduce the height of QHeader?

### DISCUSSION THREAD

**Discussion by Ammamon** [quasarframework/quasar#11243] [General - Components / Directives / etc] [answered]
Is it possible to reduce the height of QHeader?  Not it looks too bulky and occupying a good amount of space.

**Comment by rstoenescu** [comment, marked as answer]
According to Material Design specs it needs to have a min-height, but you can change it in your css.

**Reply by Ammamon** [reply]
Thank you.
I added the following style and it worked, though I had to remove the background color from toolbar.

```
<style scoped>
.headerCustomStyle {
  height: 30px !important;
  display: flex;
  align-items: center !important;
}
</style>
```
